src/Illuminate/Container/Container.php

```diff
@@ -679,15 +686,16 @@ protected function getConcrete($abstract)
     }
 
     /**
-     * Get the contextual concrete binding for the given abstract.
+     * Get the contextual concrete binding for the given concretion and abstract.
      *
+     * @param  string  $concrete
      * @param  string  $abstract
      * @return string|null
      */
-    protected function getContextualConcrete($abstract)
+    public function getContextualConcrete($concrete, $abstract)
```

Comment:
Not sure if it's something we really need to worry about. It's unlikely anyone is messing with that method.

Ping @taylorotwell.